### PR TITLE
add prettier & yarn command `yarn run fmt`

### LINKED ISSUE #2090: add prettier
the first step for #2081 

- [ ] add prettier to devDependencies
- [ ] set up prettier with some reasonable conventions
- [ ] create a new npm script called "fmt" that runs prettier

### PR DESCRIPTION
**Pull request by bnchdrff**
closes #2090 

see #2081 for the backstory

also, remove the existing precommit hooks until we fix them -- are ppl ok with this?

### CODE REVIEW

File: .prettierrc [hunk 1]
```diff
@@ -0,0 +1,5 @@
+{
+    tabWidth: 4,
+    singleQuote: true,
+    trailingComma: "es5"
```
Comment (goldibex) [inline .prettierrc]: Let's use JSON for this file, as that's already the format for .babelrc and .eslintrc

File: package.json [hunk 2]
```diff
@@ -23,6 +24,7 @@
   "license": "MIT",
   "dependencies": {
     "@steem/crypto-session": "git+https://github.com/steemit/crypto-session.git#83a90b319ce5bc6a70362d52a15a815de7e729bb",
+    "@steemit/steem-js": "0.6.7",
```
Comment (goldibex) [inline package.json]: this shouldn't be here?

**Comment by goldibex** [comment]
and yes, removing the existing precommit hooks is fine for the moment

File: package.json (hunk 2 above)
Comment (bnchdrff) [reply package.json]: ah it got re-ordered by yarn

**Pull request merged**